go-controller/pkg/ovn/gress_policy.go

```diff
@@ -0,0 +1,369 @@
+package ovn
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	knet "k8s.io/api/networking/v1"
+	"k8s.io/klog"
+)
+
+type gressPolicy struct {
+	policyNamespace string
+	policyName      string
+	policyType      knet.PolicyType
+	idx             int
+
+	// peerAddressSets points to all the addressSets that hold
+	// the peer pod's IP addresses. We will have one addressSet for
+	// local pods and multiple addressSets that each represent a
+	// peer namespace
+	peerAddressSets map[string]bool
+
+	// sortedPeerAddressSets has the sorted peerAddressSets
+	sortedPeerAddressSets []string
```

Comment:
feels like this could be refactored away, but not necessary in this PR.

Reply:
@squeed it's changed a bit in the AddressSets PR but we need to sort the sets to make sure the address order is stable when we replace it wholesale. However, it's less used now that we add/remove individual addresses so maybe we dont' actually care anymore since replace operations are very rare.